frontend/src/components/SystemSubmitDrawer/index.tsx

```diff
@@ -222,6 +259,7 @@ export function SystemSubmitDrawer(props: Props) {
       }
     } finally {
       setState(State.other);
+      resetAllFormFields();
```

Comment:
I think we should maybe only reset form items when the request is successful. What do you think?